frontend/src/component/common/Search/SearchSuggestions/SearchSuggestions.tsx

```diff
@@ -69,16 +71,19 @@ export const SearchSuggestions: VFC<SearchSuggestionsProps> = ({
                 getColumnValues(column, row)
             );
 
+            const options = [...new Set(filterOptions)]
+                .filter(Boolean)
+                .flatMap(item => item.split('\n'))
+                .filter(item => !item.includes('"') && !item.includes("'"))
+                .map(quote)
+                .sort((a, b) => a.localeCompare(b));
+
             return {
                 name: column.filterName,
                 header: column.Header ?? column.filterName,
-                options: [...new Set(filterOptions)]
-                    .filter(Boolean)
-                    .flatMap(item => item.split('\n'))
-                    .map(item => (item.includes(' ') ? `"${item}"` : item))
-                    .sort((a, b) => a.localeCompare(b)),
+                options,
                 suggestedOption:
-                    filterOptions[randomRow] ?? `example-${column.filterName}`,
+                    options[randomRow] ?? `example-${column.filterName}`,
```

Comment:
we want to suggest from the valid options after pre-processing. E.g. with quotes, with split new line